src/router/index.js

```diff
@@ -0,0 +1,28 @@
+import Vue from 'vue';
+import Home from '../views/Home.vue';
+
+Vue.use(VueRouter);
```

Comment:
The next exception is:

> Uncaught ReferenceError: VueRouter is not defined

it isn't intended to be reviewed, don't it?

Reply:
Thanks, looks like I need to pass there a real router.

like 
```js
const router = new VueRouter({})
Vue.use(router);
```

Right?

Reply:
@davidyuk I will research more

Reply:
> If using a module system (e.g. via vue-cli), import Vue and VueRouter and then call `Vue.use(VueRouter)`.

https://router.vuejs.org/guide/#javascript